packages/venia-ui/lib/components/OrderHistoryPage/OrderDetails/item.js

```diff
@@ -0,0 +1,70 @@
+import React, { useMemo } from 'react';
+
+import { Link, resourceUrl } from '@magento/venia-drivers';
+import { mergeClasses } from '@magento/venia-ui/lib/classify';
+
+import Button from '../../Button';
+import ProductOptions from '../../LegacyMiniCart/productOptions';
+import Image from '../../Image';
+
+import defaultClasses from './item.css';
+
+const Item = props => {
+    const {
+        product_name,
+        product_sale_price,
+        quantity_ordered,
+        selected_options,
+        thumbnail,
+        url_key,
+        url_suffix
+    } = props;
+    const { url } = thumbnail;
```

Comment:
We saw an error from this LOC when removing `apollo-cache-persist` and refreshing the page when on the order history page.

Worth investigating.